discgolfbot/pdga/pdgaPlayer.py

```diff
@@ -0,0 +1,13 @@
+
+class PdgaPlayer:
```

Comment:
Foreslår å endre denne slik at du kan bruke .__dict__ isteden;
```suggestion
class PdgaPlayer:
    def __init__(self, pdga_number, player_name, discord_id):
        self.pdga_number = pdga_number
        self.player_name = player_name        
        self.discord_id = discord_id
```

Reply:
> Foreslår å endre denne slik at du kan bruke .**dict** isteden;

Nå forstår jeg ikke hva du mener?

Reply:
Eksempel:

Klassen score.name:
```python
class Name:
    """Name class, contain name and aliases"""
    def __init__(self, name:str, alias:list=None):
        self.name = name
        self.alias = alias

    @staticmethod
    def from_json(json_object):
        """Returns Name object from json object"""
        return Name(json_object['name'], json_object['alias'])


# Istedenfor å lage en custom `__to_dict__` kan, man bruke `.__dict__` direkte 🙂
# Testen som konverterer json objekt til Name objekt:
def test_alias_name_serialize_deserialize():
    """Test to create a Name, serialize it and deserialize"""
    player_name = Name("uDisc name 2",["alias1", "alias2", "etc"])
    json_file = json.dumps(player_name.__dict__)
    assert json_file is not None
    json_object = json.loads(json_file)
    assert json_object is not None
    player_name_copy = Name.from_json(json_object)
    assert player_name == player_name_copy```